src/components/VmDetails/cards/SnapshotsCard/SnapshotItem.js

```diff
@@ -22,6 +22,19 @@ import { PendingTaskTypes } from '../../../../reducers/pendingTasks'
 
 const MAX_DESCRIPTION_SIZE = 50
 
+const SnapshotAction = ({ children, className, disabled, onClick }) => {
+  return <a onClick={!disabled ? onClick : null} className={`${className} ${disabled && 'disabled'}`}>
```

Comment:
You used `null` here for the onClick but `undefined` in another file.  It would be nice to stay consistent.  Could still do `{!disabled && onClick}` and let it default to undefined by itself.

Reply:
Er, nope.  Still onClick still needs a function.  Will use `noop`.